.drone.yml

```diff
@@ -0,0 +1,18 @@
+kind: pipeline
+name: test on arm64
+
+platform:
+  arch: arm64
+
+steps:
+- name: test
+  image: quay.io/condaforge/mambaforge:latest
```

Comment:
By the way, it is very nice to use this docker image with mambaforge included! :)

Reply:
Running daily sounds good.

Yes, mambaforge is a breezy of fresh air. This weekend I switched from a broken miniforge installation to mambaforge and I am not looking back, By the way, what broke my miniconda installation was uninstalling mamba_gator. I uninstalled it because it was using a lot of resources every time that I was opening jupyterlab.

Reply:
> By the way, what broke my miniconda installation was uninstalling mamba_gator. I uninstalled it because it was using a lot of resources every time that I was opening jupyterlab.

Indeed, this is now fixed in the release version - see https://github.com/mamba-org/gator/pull/133.

There is still a bug with uninstalling and reinstalling `gator`... which I think I was able to reproduce.

Reply:
> There is still a bug with uninstalling and reinstalling `gator`... which I think I was able to reproduce.

Now is now [fixed](https://github.com/mamba-org/gator/pull/140) in gator 5.1.1.